downstream/modules/platform/proc-controller-set-up-SAML.adoc

```diff
@@ -0,0 +1,384 @@
+----
+{
+ "remove": false,
+ "team_org_map": [
+  {
+   "team": "internal:unix:domain:admins",
+   "organization": "Default",
+   "team_alias": "Administrators"
+  },
+  {
+   "team": "Domain Users",
+   "organization_alias": "OrgAlias",
+   "organization": "Default"
+  }
+ ],
+ "saml_attr": "member-of"
+}
+----
++
+Once the user authenticates, {ControllerName} creates organization and team aliases.
++
+. Optional: Provide team membership mapping in the *SAML Team Map* field. 
+For more information, see [ADD XREF]Organization and Team Mapping.
+. Optional: Provide security settings in the *SAML Security Config* field. 
+This field is the equivalent to the `SOCIAL_AUTH_SAML_SECURITY_CONFIG` field in the API. 
+For more information, see link:https://github.com/SAML-Toolkits/python-saml#settings[OneLogin's SAML Python Toolkit].
++
+{ControllerNameStart} uses the `python-social-auth` library when users log in through SAML. 
+This library relies on the `python-saml` library to make the settings available for the next two optional fields, *SAML Service Provider extra configuration data* and *SAML IDP to extra_data attribute mapping*.
++
+* The *SAML Service Provider extra configuration data* field is equivalent to the `SOCIAL_AUTH_SAML_SP_EXTRA` in the API. 
+For more information, see link:https://github.com/SAML-Toolkits/python-saml#settings[OneLogin's SAML Python Toolkit] to learn about the valid service provider extra (`SP_EXTRA`) parameters.
+* The *SAML IDP to extra_data attribute mapping* field is equivalent to the `SOCIAL_AUTH_SAML_EXTRA_DATA` in the API. 
+For more information, see Python's SAML link:https://python-social-auth.readthedocs.io/en/latest/backends/saml.html#advanced-settings [Advanced Settings] documentation.
+* The *SAML User Flags Attribute Mapping* field enables you to map SAML roles and attributes to special user flags. 
+The following attributes are valid in this field:
+** `is_superuser_role`: Specifies one or more SAML roles which grants a user the superuser flag.
+** `is_superuser_attr`: Specifies a SAML attribute which grant a user the superuser flag.
+** `is_superuser_value`: Specifies one or more values required for `is_superuser_attr` that is required for the user to be a superuser.
+** `remove_superusers`: Boolean indicating if the superuser flag should be removed for users or not. 
+This defaults to `true`. 
+** `is_system_auditor_role`: Specifies one or more SAML roles which will grant a user the system auditor flag
+** `is_system_auditor_attr`: Specifies a SAML attribute which will grant a user the system auditor flag
+** `is_system_auditor_value`: Specifies one or more values required for `is_system_auditor_attr` that is required for the user to be a system auditor.
+** `remove_system_auditors`: Boolean indicating if the `system_auditor` flag should be removed for users or not. 
+This defaults to `true`. 
++
+The `role` and `value` fields are lists and are _or_ logic. 
```

Comment:
```suggestion
The `role` and `value` fields are lists that use `OR` logic. 
```
it might be best to use a code format for `or` to distinguish it from the word "or"